libs/npm/core/src/hooks/api-subscriber/useApiSubscriberContents.ts

```diff
@@ -0,0 +1,81 @@
+import { AxiosResponse } from 'axios';
+import React from 'react';
+
+import {
+  IContentFilter,
+  IContentListModel,
+  IContentModel,
+  ILifecycleToasts,
+  IPaged,
+  defaultEnvelope,
+  useApi,
+} from '..';
+import { toQueryString } from '../../utils';
+
+/**
+ * Common hook to make requests to the API.
+ * @returns CustomAxios object setup for the API.
+ */
+export const useSubscriberApiContents = (
+  options: {
+    lifecycleToasts?: ILifecycleToasts;
+    selector?: Function;
+    envelope?: typeof defaultEnvelope;
+    baseURL?: string;
+  } = {},
+) => {
+  const api = useApi(options);
+
+  return React.useRef({
+    findContent: (filter?: IContentFilter) => {
+      const params = {
+        ...filter,
+        actions: filter?.actions?.length ? filter.actions : undefined,
+      };
+      return api.get<IPaged<IContentModel>, AxiosResponse<IPaged<IContentModel>>, any>(
+        `/subscriber/contents?${toQueryString(params)}`,
+      );
+    },
+    getContent: (id: number) => {
+      return api.get<IContentModel | undefined, AxiosResponse<IContentModel | undefined>, any>(
+        `/subscriber/contents/${id}`,
+      );
+    },
+    updateContent: (content: IContentModel) => {
```

Comment:
Subscribers cannot update content

Reply:
I vaguely remember seeing on Bobbi's mockup a way to edit a headline.. I may be mistaken

Reply:
Subscribers cannot edit any content.

When creating reports they can edit the report data only.  We don't currently have a way to do this.